FAIL
Kernel panic in run: PANIC at ../../userprog/exception.c:97 in kill(): Kernel bug - unexpected interrupt in kernel
Call stack: 0x80042183df 0x800421d163 0x800421d2e2 0x80042091b7 0x80042095d5 0x8004217e12 0x800421c1a6 0x800421d8b3 0x800421d537 0x800421d357
Translation of call stack:
0x00000080042183df: debug_panic (lib/kernel/debug.c:32)
0x000000800421d163: kill (userprog/exception.c:103)
0x000000800421d2e2: page_fault (userprog/exception.c:160 (discriminator 12))
0x00000080042091b7: intr_handler (threads/interrupt.c:352)
0x00000080042095d5: intr_entry (threads/intr-stubs.o:?)
0x0000008004217e12: strlcpy (lib/string.c:314)
0x000000800421c1a6: process_exec (userprog/process.c:251 (discriminator 3))
0x000000800421d8b3: exec (userprog/syscall.c:159)
0x000000800421d537: syscall_handler (userprog/syscall.c:78)
0x000000800421d357: no_sti (userprog/syscall-entry.o:?)
